robot/rover/MobilePlatform/MobilePlatform.ino

```diff
@@ -215,6 +229,22 @@ void loop() {
         }
         sinceFeedbackPrint = 0;
     }
+    
+    if(Cmds.isLedOn){ // Checks to see if indicator LED is currently on
+        if(Cmds.indicatorRgbArray[1] == 255){ // Checks to see if 'eureka mode' is running
+          if(sinceEurekaToggle > LED_BLINK_INTERVAL){ // Set LEDs ON after 1000ms  
+            setIndicatorMode(activityIndicator.Color(Cmds.indicatorRgbArray[0],Cmds.indicatorRgbArray[1],Cmds.indicatorRgbArray[2]));
+          }
+          if(sinceEurekaToggle > EUREKA_LED_INTERVAL){ // Set LEDs OFF after 2000ms
+            setIndicatorMode(activityIndicator.Color(0,0,0)); // Code to turn off all LEDs by setting RGB values to 0
+            sinceLedToggle = 0; // Reset blink timer
+          }
+        }
+        setIndicatorMode(activityIndicator.Color(Cmds.indicatorRgbArray[0],Cmds.indicatorRgbArray[1],Cmds.indicatorRgbArray[2])); // Set indicator LED RGB values to requested values
+    }
+    else{
+        setIndicatorMode(activityIndicator.Color(0,0,0)); // Set LEDs OFF
```

Comment:
Okay, now that you put your comments i have a better idea. I would maybe put a comment somewhere in the beginning of this code block explaining that part of this block of logic is to handle the situation that we were talking about, that the code needs to be smart about how it switches modes when it's in eureka mode, because of the fact that there's no on/off status in the library so you have to use this stuff